virttest/data_dir.py

```diff
@@ -18,28 +19,14 @@
 
 from six.moves import xrange
 
-if hasattr(sys, 'real_prefix'):
-    # unlike default execution venv prefix does not contain /usr
-    _DEFAULT_SHARED_PATH = os.path.join(sys.prefix, "shared")
-else:
-    _DEFAULT_SHARED_PATH = os.path.join(sys.prefix, "share",
-                                        "avocado-plugins-vt",
-                                        "shared")
-
-if (os.path.isdir(_DEFAULT_SHARED_PATH) and
-        len(os.listdir(_DEFAULT_SHARED_PATH)) > 0):
-    _ROOT_PATH = os.path.dirname(_DEFAULT_SHARED_PATH)
-else:
-    _ROOT_PATH = os.path.realpath(os.path.dirname(os.path.dirname(__file__)))
-
-ROOT_DIR = os.path.abspath(_ROOT_PATH)
-BASE_BACKEND_DIR = os.path.join(ROOT_DIR, 'backends')
-DATA_DIR = os.path.join(data_dir.get_data_dir(), 'avocado-vt')
-SHARED_DIR = os.path.join(ROOT_DIR, 'shared')
-DEPS_DIR = os.path.join(ROOT_DIR, 'shared', 'deps')
+BASE_BACKEND_DIR = pkg_resources.resource_filename('virttest', 'backends')
```

Comment:
I am really glad to see this change, Avocado-vt was waiting for it way too long. It will require a lot of testing, though...

Reply:
I am doing some large scale testing of our test suite and everything works well there so I assume follow-up errors will be restricted to tests and thus test suites mostly.